server/routes.go

```diff
@@ -1482,11 +1483,22 @@ func (s *Server) ChatHandler(c *gin.Context) {
 		return
 	}
 
+	slog.Debug("chat request", "images", len(images), "prompt", prompt)
```

Comment:
This was removed upstream so there's something up with your rebase

Reply:
Ahh okay I had added back in when rebasing. Will rm